kickstarter/android-oss
# NT-1574: Android Add-ons Pills Polish

# 📲 What

Update padding between pills on the Add-ons screen to match the padding between pills on the Reward cards.

# 🤔 Why

Design parity.

# 🛠 How

Small layout adjustments

# 👀 See

| Before 🐛 | After 🦋 |
![Screenshot_20201007-095424](https://user-images.githubusercontent.com/19390326/96177363-c6cf8100-0efb-11eb-96ef-ad99cb1a164b.jpeg) ![Screenshot_1602790212](https://user-images.githubusercontent.com/19390326/96177375-ca630800-0efb-11eb-867e-5f8f318eac13.png)


# 📋 QA

- Select project with add-ons
- Tap "Back this project"
- Select reward with add-ons
- Look for add-on with several pills to observe padding inbetween

# Story 📖

[NT-1574: Android Add-ons Pills Polish](https://kickstarter.atlassian.net/browse/NT-1574)
